source/analytics/metabase/metabase_helm_charts/charts/metabase/values_sandbox.yaml

```diff
@@ -0,0 +1,269 @@
+# Currently Metabase is not horizontly scalable. See
+# https://github.com/metabase/metabase/issues/1446 and
+# https://github.com/metabase/metabase/issues/2754
+# NOTE: Should remain 1
+replicaCount: 1
+
+pdb:
+  create: false
+  minAvailable: 1
+  maxUnavailable: ""
+
+deploymentAnnotations: {}
+deploymentLabels: {}
+podAnnotations: {}
+podLabels: {}
+image:
+  repository: metabase/metabase
+  tag: v0.46.6
+  command: []
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+## String to fully override metabase.fullname template
+##
+# fullnameOverride:
+
+# Override MAX Pool connections defaults
+extraenv: 
+  app_db_max_conn_pool: 100
```

Comment:
@sralla-eq for the dev specify the default value, and change this to 50 for now.